### DISCUSSION TITLE
How to allow read project members?

### DISCUSSION THREAD

**Discussion by iBobik** [contember/admin#573] [Support]
I would like to content_admin allow to read person e-mails. How to do it?

I tried this query:
```
fragment ProjectMember on ProjectIdentityRelation {
  identity {
    person {
      id
      email
    }
  }
  memberships { role }
}

query getProjectMembers ($projectSlug: String!) {
  project: projectBySlug(slug: $projectSlug) {
    members {
      ...ProjectMember
    }
  }
}
```

When called by the admin key it returns what I need, but when called by content_admin permission members array is empty. I guess it is because of permission. (It would be better to return permission error than empty content.)

**Comment by matej21** [comment]
Hello, 
In the tenant section of the role ACL, you have the option to specify which roles it [can manage](https://docs.contember.com/reference/engine/schema/acl#manage-permissions). However, it does not provide the flexibility to allow only "view" permissions; you need to grant the "manage" permission in its entirety. Furthermore, you need to specify all the roles individually, you cannot tell that this role can manage everyone.
